src/c21server/work_server/work_server.py

```diff
@@ -44,8 +44,15 @@ def _put_results():
         print("job_id: %s, value: %s" % (key, data[key]))
         return '', 200
 
-    return workserver
+    @server.app.route('/get_client_id', methods=['GET'])
```

Comment:
This decorator should be `@workserver.app.route('/get_client_id', methods=['GET'])`